js/localization.js

```diff
@@ -17,24 +17,23 @@ function t(key, params) {
 }
 
 
-function tE(id, key, attr, esc) {
-  if (attr) {
-    document.getElementById(id).setAttribute(attr, t(key));
-  } else if (typeof esc === 'undefined' || esc) {
-    document.getElementById(id).appendChild(document.createTextNode(t(key)));
-  } else {
-    document.getElementById(id).innerHTML = t(key);
+function tHTML(html, tag) {
```

Comment:
Judging by AMO [retarded] rules you'll have to rework all HTML string construction properly (completely) into DOM/$element and delete this function.